docs/README.md

```diff
@@ -96,6 +96,14 @@ optional arguments:
   --nsfw                Include checking of NSFW sites from default list.
 ```
 
+In case installation fails:
+```bash
+python -m sherlock_project user1 user2 user3 
+```
+or
+```bash
+python3 -m sherlock_project user1 user2 user3
+```
```

Comment:
Could you explain this section please?

Python's gotta get it from somewhere, and that would presumably be by installation (which either failed or didn't)

Reply:
I'm not 100% sure why only this method works for me, but I'm guessing even if the installation is somewhat successful, running it by directly calling the module with Python bypasses permission or setup issues.